downstream/modules/platform/ref-postgresql-requirements.adoc

```diff
@@ -0,0 +1,26 @@
+[id="ref-postgresql-requirements"]
+
+== PostgreSQL requirements
+
+{PlatformName} uses PostgreSQL 13.
+
+* PostgreSQL user passwords are hashed with SCRAM-SHA-256 secure hashing algorithm before storing in the database.
+* Since {ControllerName} and {HubName} use a Software Collections version of PostgreSQL in 3.8, the `rh-postgresql10` scl must be enabled to access the database. 
```

Comment:
IBMSG "Since": To facilitate translation, use this word only to refer to time. Do not use it as a synonym for "because".

To avoid passive voice here "the `rh-postgresql10` scl must be enabled to access the database. " >> "you must enable the `rh-postgresql10` scl to access the database.